<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CounsellorForm</class>
 <widget class="QMainWindow" name="CounsellorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="spacing">
     <number>8</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblAccountID">
        <property name="text">
         <string>Account ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblClass">
        <property name="text">
         <string>Class: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblStudentCount">
        <property name="text">
         <string>Total Student: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblAverage">
        <property name="text">
         <string>Average: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblGreatStudent">
        <property name="text">
         <string>Excellent And Good: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="acQuery"/>
    <addaction name="acQueryByGrade"/>
    <addaction name="acStatistics"/>
    <addaction name="acExport"/>
    <addaction name="separator"/>
    <addaction name="acLogout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acStatistics">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="acExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="acLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="acQuery">
   <property name="text">
    <string>Query By Name</string>
   </property>
  </action>
  <action name="acQueryByGrade">
   <property name="text">
    <string>Query By Grade</string>
   </property>
  </action>
  <action name="actionSort_By_Grade">
   <property name="text">
    <string>Sort By Grade</string>
   </property>
  </action>
  <action name="actionSort_By_ID">
   <property name="text">
    <string>Sort By ID</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
